src/patternfly/components/Table/table-grid.scss

```diff
@@ -130,28 +134,65 @@
 
 // - Table mobile layout
 @include pf-mobile-layout {
-  --#{$table}--cell--PaddingBlockStart: var(--#{$table}--m-grid--cell--PaddingBlockStart);
-  --#{$table}--cell--PaddingInlineEnd: var(--#{$table}--m-grid--cell--PaddingInlineEnd);
-  --#{$table}--cell--PaddingBlockEnd: var(--#{$table}--m-grid--cell--PaddingBlockEnd);
-  --#{$table}--cell--PaddingInlineStart: var(--#{$table}--m-grid--cell--PaddingInlineStart);
-
-  // - Table favorite
-  --#{$table}__favorite--c-button--MarginBlockStart: auto;
-  --#{$table}__favorite--c-button--MarginInlineEnd: auto;
-  --#{$table}__favorite--c-button--MarginBlockEnd: auto;
-  --#{$table}__favorite--c-button--MarginInlineStart: auto;
```

Comment:
`.#{$table}__favorite` is part of the standardized, "action item" group. Specific `margin` settings are no longer necessary.